### PR TITLE
Synchronize all canUseTopic headers to prevent runtimes

### PR DESCRIPTION
**Pull request by optimumtact**
TODO: go through and check all instances of use to ensure they properly specify which positional arguments to use

### CODE REVIEW

File: code/game/gamemodes/devil/true_devil/_true_devil.dm [hunk 1]
```diff
@@ -92,7 +92,7 @@
 /mob/living/carbon/true_devil/IsAdvancedToolUser()
 	return 1
 
-/mob/living/carbon/true_devil/canUseTopic(atom/movable/M, be_close = 0)
+/mob/living/carbon/true_devil/canUseTopic(atom/movable/M, be_close = 0, no_dextery = 0)
```
Comment (Iamgoofball) [inline code/game/gamemodes/devil/true_devil/_true_devil.dm]: 🎓

**Review by Iamgoofball — CHANGES_REQUESTED** [review]
# oranges cant spell dexterity

**Comment by optimumtact** [comment]
git grep -l '/canUseTopic' | xargs sed -i 's#/canUseTopic(.*)#/canUseTopic(atom/movable/M, be_close = 0, no_dextery = 0)#

Also @Iamgoofball I can't change that, it's a legacy setting

**Review by Iamgoofball — APPROVED** [review]
# ignore previous review

# oranges can spell dexterity some oldcoder fucked it up

# i bet it was uhangi

**Comment by optimumtact** [comment]
I figure that since I"m going to have to audit every canUseTopic anyway I can fix it up now

**Comment by optimumtact** [comment]
will fix this, this weekend

**Comment by optimumtact** [comment]
Maybe this next weekend for sure

**Comment by Iamgoofball** [comment]
like slips?

**Comment by KorPhaeron** [comment]
stale, reopen it when you're ready

**Comment by optimumtact** [comment]
actually not yet

**Pull request closed without merging**